demo/example.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+    <title>kwc-auth demo</title>
+    <style>
+        body {
+            margin: 0;
+        }
+    </style>
+</head>
+
+<body>
+    <kwc-auth-example></kwc-auth-example>
+</body>
+<script type="module">
+    import '../dist/kwc-auth-example.js';
+    import { Actions } from '../dist/examples/actions.js';
+    import { viewsArray } from '../dist/examples/viewsArray.js';
```

Comment:
dash-case?

Reply:
Might be nitpicking, but while the API is still under develpoment, maybe we could find a better name for `viewsArray` Either something simple like `views` so that people integrating get a clean API. Or something more explicit like `flowDefinition`